core/tests/ts-integration/tests/system.test.ts

```diff
@@ -30,6 +30,26 @@ describe('System behavior checks', () => {
         alice = testMaster.mainAccount();
     });
 
+    test('Network should be supporting Cancun+Deneb', async () => {
```

Comment:
how does this test that we have upgraded to cancun+deneb? i.e. why does the first transaction fail but the second passes

Reply:
We want to check that [`POINT_EVALUATION_PRECOMPILE_ADDRESS`](https://eips.ethereum.org/EIPS/eip-4844#point-evaluation-precompile) actually contains a precompile and would reject an empty input.

Reply:
makes sense!